resources/dicts/events/ceremonies/ceremony-master.json

```diff
@@ -2789,6 +2789,375 @@
+  ],
+  "warrior_70": [
+    [
+      "warrior",
+      "prepared",
+      "general_mentor",
+      "alive2_parents",
+      "yes_leader",
+      "general_backstory",
+      "charismatic",
+      "ambitious",
+      "confident",
+      "childish",
+      "righteous",
+      "arrogant",
+      "competitive",
+      "flamboyant"
+    ],
+    "(old_name) has thrown {PRONOUN/m_c/self} at every challenge in {PRONOUN/m_c/poss} path, and l_n notes {PRONOUN/m_c/subject} {VERB/m_c/have/has} the r_h of a tiger, may it serve {PRONOUN/m_c/object} well. It's time for (old_name) to take both {PRONOUN/m_c/poss} place as an adult of c_n, and {PRONOUN/m_c/poss} full name of m_c."
+  ],
+  "warrior_71": [
+    [
+      "warrior",
+      "prepared",
+      "general_mentor",
+      "general_parents",
+      "yes_leader",
+      "general_backstory",
+      "oblivious",
+      "calm",
+      "wise"
+    ],
+    "As the Clan yowls m_c's new warrior name {PRONOUN/m_c/subject} {VERB/m_c/smile/smiles} and {VERB/m_c/nod/nods}, pleased but not overcome. l_n jumps down to congratulate {PRONOUN/m_c/object}. As their heads press together, l_n whispers that balanced sense of self and r_h will serve m_c well."
+  ],
+  "warrior_72": [
+    [
+      "warrior",
+      "prepared",
+      "general_mentor",
+      "general_parents",
+      "yes_leader",
+      "general_backstory",
+      "strict",
+      "responsible"
+    ],
+    "With c_n gathered for celebration, l_n highlights (old_name)'s r_h, {PRONOUN/m_c/poss} dedication to truth and the warrior code. It is for this that l_n names {PRONOUN/m_c/object} m_c, welcoming {PRONOUN/m_c/object} as a full warrior of c_n."
+  ],
+  "warrior_73": [
+    [
+      "warrior",
+      "prepared",
+      "alive_mentor",
+      "general_parents",
+      "yes_leader",
+      "general_backstory",
+      "charismatic",
+      "responsible",
+      "careful"
+    ],
+    "The poise, decorum, and r_h (old_name) acts with are a credit to c_n, l_n meows. (old_name) tries to live up to such pretty words, but all {PRONOUN/m_c/subject} can focus on are the ringing yowls of m_c as {PRONOUN/m_c/poss} new warrior name echos around camp."
+  ],
+  "warrior_74": [
+    [
+      "warrior",
+      "prepared",
+      "alive_mentor",
+      "general_parents",
+      "yes_leader",
+      "general_backstory",
+      "childish",
+      "oblivious",
+      "insecure"
+    ],
+    "l_n calls c_n to attention. They are gathered here to celebrate the hard work, persistence, and r_h of c_n's newest warrior. (old_name)'s jaw drops as {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} summoned forward, and {PRONOUN/m_c/subject} {VERB/m_c/sit/sits} , disbelieving but thrilled, as l_n grants {PRONOUN/m_c/object} the name m_c and cheers break out."
```

Comment:
{PRONOUN/m_c/subject} {VERB/m_c/sit/sits} ,

there shouldnt be a space between the sit verb tag and the comma